ntcore/src/generate/main/java/NetworkTableInstance.java.jinja

```diff
@@ -49,11 +49,8 @@ public final class NetworkTableInstance implements AutoCloseable {
     /** Running in server mode. */
     kServer(0x01),
 
-    /** Running in NT3 client mode. */
-    kClient3(0x02),
-
-    /** Running in NT4 client mode. */
-    kClient4(0x04),
+    /** Running in client mode. */
+    kClient(0x04),
```

Comment:
Was it intentional to keep `kClient` at the same value (which means `0x02` is unused)?